### ISSUE TITLE
Base Namespace should be Calcinai/XeroPHP

### ISSUE THREAD

**Issue by mhlavac** [calcinai/xero-php#45]
The base namespace for everything should be Calcinai/XeroPHP.

Unfortunately this is bigger change in the code and I don't won't to do it before someone says it's good thing to do.

It also needs to go to version 2.x, because there will be major BC break.

**Comment by judgej** [comment]
Calcinai\XeroPHP ;-)

**Comment by mhlavac** [comment]
@judgej :+1:

**Comment by calcinai** [comment]
I like the idea of this, but what would be better still is maybe transferring ownership to an organization if we're going through the trouble of renaming it?

**Comment by judgej** [comment]
Yeah. why not. I created an organisation "academe" in github, named after my company, and put loads of stuff in there. The org then feeds into other areas such as the composer vendor namespace. I'm assuming this is what you mean, and not a legal entity.

Where you thinking of an org generic enough to cover other packages you may write?

**Comment by mhlavac** [comment]
@calcinai @judgej the best organisation to own this would be Xero, but I am not sure it's wise ;-). I think for now it's fine if it uses your name. You will get some recognition for what you've done + you can always move it somewhere else without changing PHP namespace.

I think change of namespace is not such a big problem. If you ever want to do it then everyone has to do search & replace Calcinai\XeroPHP -> WhatEver\XeroPHP and it's done.

**Comment by calcinai** [comment]
Funny you mention Xero, I think that would definitely be the most appropriate place for this code, but then again.. 

But yeah, we'll see what happens and address in a 2.0 release. I have a WIP to split this all into two repositories anyway, one that generates a _real_ set of XSD or (swagger.json)[http://swagger.io] files and another that parsed those and builds the models.  Can all be part of that refactor.

**Comment by gerrywastaken** [comment]
@calcinai @mhlavac Don't even joke about handing over ownership to them, they clearly have management there at the moment who have Xero experience running a software company.

You definitely deserve the credit Michael for the tremendous amount of work you have put in. Keep it under your name. A lot of people no doubt considered rewriting XeroOAuth-PHP (I know I did), but you did it and did it well.

**Comment by calcinai** [comment]
Was thinking about this again.  I'm going to get started on the spec generator and want somewhere to put it..  How about a 'xero-community' organisation?

**Comment by gerrywastaken** [comment]
That's a fantastic name, so long as we allow non PHP xero community projects. I wouldn't be surprised if other languages were running into the issues with Xero that we are. Also given those linters need full account access it wouldn't be a bad idea to separate it out from your account.

**Comment by coop182** [comment]
Could also consider name like Accounting-APIs - frustration with accounting API's is certainly not limited to just Xero... MYOB ARL and Intuit QBO API's both leave a lot to be desired and it would be great to eventually give them a similar treatment.
